test/NuGet.Core.Tests/NuGet.ProjectModel.Test/LockFileDependencyComparerWithoutContentHashTests.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.ProjectModel.ProjectLockFile;
+using NuGet.ProjectModel.Test.Builders;
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.ProjectModel.Test
+{
+    public class LockFileDependencyComparerWithoutContentHashTests
+    {
+        [Fact]
+        public void NullReferencesAreEqual()
```

Comment:
A better test naming pattern is:

    <NameOfMemberUnderTest>_<Scenario>_<ExpectedResult>

So this test would be:

    Equals_WhenBothArgumentsAreNull_ReturnsTrue